<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>host_widget</class>
 <widget class="QWidget" name="host_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Molo</string>
  </property>
  <widget class="QLineEdit" name="host_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>http://</string>
   </property>
   <property name="placeholderText">
    <string>Kitsu server address</string>
   </property>
  </widget>
  <widget class="QLabel" name="host_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_msg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
